packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -118,21 +118,8 @@ class FlutterDevice {
     // used to file a bug, but the compiler will still start up correctly.
     if (targetPlatform == TargetPlatform.web_javascript) {
       // TODO(zanderso): consistently provide these flags across platforms.
-      final String platformDillName;
+      const String platformDillName = 'ddc_outline_sound.dill';
```

Comment:
Is there cleanup somewhere to remove the unsound .dill file from the packaged flutter release? Not essential but will cut down the overall download size.

Reply:
That's a good point, I'll file a tracking issue for this.

Reply:
https://github.com/flutter/flutter/issues/122215